docs/contribution/get-started.mdx

```diff
@@ -0,0 +1,67 @@
+# Contributing to Widgetbook
+
+We appreciate your interest in contributing to Widgetbook. Whether it's through fixing bugs, enhancing documentation, or suggesting new features, your efforts are greatly valued.
+
+The following are some guidelines for contributing to Widgetbook and its packages. Remember, these are primarily guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Proposing a Change
+
+If you plan to modify the public API or make non-trivial changes to the implementation, we recommend [filing an issue](https://github.com/widgetbook/widgetbook/issues) first. This helps us align your proposal before you put significant effort into it.
+
+You can submit a pull request for bug fixes immediately, but we still recommend filing an issue detailing what you're fixing. This is useful if we don't accept that specific fix but want to track the issue.
+
+## Contribution Workflow
+
+The following steps will guide you through the contribution process:
+
+### Preparation
+
+1. **Fork the Repository:** Start by forking the Widgetbook repository and creating your own branch from the `main` branch.
+
+2. **Install Dependencies:** Ensure your environment is ready by installing all necessary dependencies using either `flutter packages get` or `pub get`.
```

Comment:
Actually, it's likely better to run `melos bs` to setup the repository and fetch flutter packages